src/Vm.sol

```diff
@@ -128,7 +128,7 @@ interface VmSafe {
     function load(address target, bytes32 slot) external view returns (bytes32 data);
 
     // Signs data
-    function sign(string calldata privateKey, bytes32 digest) external pure returns (bytes calldata sig);
+    function sign(string memory privateKey, bytes32 digest) external pure returns (bytes memory sig);
```

Comment:
Why did we switch from calldata to memory in the argument?

Reply:
we can leave it as calldata

Reply:
Switch is back to calldata if you wish, since it is external.
But it really makes not difference since calldata casts to memory automatically, and sign is a dev function and we should not care about gas cost, it will never go into real smart contract code.